plugin-server/tests/worker/ingestion/action-manager.test.ts

```diff
@@ -137,7 +146,7 @@ describe('ActionManager', () => {
         jest.spyOn(hub.db, 'fetchAllActionsGroupedByTeam')
         jest.spyOn(hub.db, 'fetchAction')
 
-        const manager = new ActionManager(hub.postgres, { processAsyncWebhooksHandlers: false })
+        const manager = new ActionManager(hub.postgres)
```

Comment:
❓  how does this test work now

Reply:
What do you mean by that? [The ActionManager constructor](https://github.com/PostHog/posthog/blob/233e91d7ba82fba1f299b2fb1de7d0692533a028/plugin-server/src/worker/ingestion/action-manager.ts#L15-L19) only has a single param, the other was unused so I deleted it.